From 964a33c4f4653c2e44532a97c60ec601737c6d0d Mon Sep 17 00:00:00 2001
From: furrymcgee <furrymcgee@lippydanger.jumpingcrab.com>
Date: Fri, 16 Aug 2019 10:08:58 +0200
Subject: [PATCH 7/8] dpkg-genbuildinfo: error: cannot open
 /var/lib/dpkg/status: No such file or dire ctory

---
 scripts/dpkg-genbuildinfo.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/dpkg-genbuildinfo.pl b/scripts/dpkg-genbuildinfo.pl
index 827b6679f..851ef53de 100755
--- a/scripts/dpkg-genbuildinfo.pl
+++ b/scripts/dpkg-genbuildinfo.pl
@@ -92,7 +92,7 @@ sub parse_status {
     my @essential_pkgs;
 
     local $/ = '';
-    open my $status_fh, '<', $status or syserr(g_('cannot open %s'), $status);
+    open my $status_fh, '<', $status or warning(g_('cannot open %s'), $status);
     while (<$status_fh>) {
         next unless /^Status: .*ok installed$/m;
 
-- 
2.23.0.rc1

